SUI/SUI/HomeView.swift

```diff
@@ -0,0 +1,95 @@
+//  HomeView.swift
+//  SUI
+//  Created by angelina on 23.01.2023.
+
+import SwiftUI
+
+struct HomeView: View {
+    
+    // MARK: - Private Constants
+    
+    private enum Constants {
+        static let flagImageName = "flag"
+        static let emptyString = ""
+        static let propertyString = """
+                    IP Address
+                    Received
+                    Sent
+                    """
+        static let propertyValueString = """
+                    127.0.0.1
+                    0 MB
+                    0 MB
+                    """
+        static let alarmString = "Service will expire after 15 days."
+        static let basketString = "Basket"
+        static let widthImageView: CGFloat = 200
+        static let heightImageView: CGFloat = 200
+        static let cornerRadiusImageView: CGFloat = 100
+        static let basketTagNumber = 2
+        static let lineSpacingNumber: CGFloat = 20
+        static let fontSizeNumber: CGFloat = 17
+        static let paddingNumber: CGFloat = 40
+        static let kerningNumber: CGFloat = 1
+    }
+    
+    // MARK: - Public Properties
+    
+    @Binding var tabSelection: Int
+    
+    var body: some View {
+        VStack {
+            Spacer()
+            flagImageView
+            Spacer()
+            VStack {
+                Toggle(Constants.emptyString, isOn: $isOn)
```

Comment:
```suggestion
                Toggle(Constants.emptyString, isOn: $isOn)
```

убрала бы во вью модель